src/VisualStudio/Core/Def/Packaging/PackageInstallerServiceFactory.cs

```diff
@@ -487,9 +530,6 @@ private void OnWorkspaceChanged(object sender, WorkspaceChangeEventArgs e)
             if (serviceBroker == null)
                 return default;
 
-            // Make sure we are on the thread pool to avoid UI thread dependencies if external code uses ConfigureAwait(true)
-            await TaskScheduler.Default;
```

Comment:
❓ Why was this line removed?

Reply:
i'll check again, but it should be happenign higher up (and everything is CA(false) after that), so it was redundant.

Reply:
added back with comment.